frontend/src/entities/configuration/emr-configuration.tsx

```diff
@@ -0,0 +1,315 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Container,
+    ExpandableSection,
+    FormField,
+    Grid,
+    Header,
+    Input,
+    Multiselect,
+    SpaceBetween
+} from '@cloudscape-design/components';
+import React, {useEffect, useState} from 'react';
+import {ClusterTypeConfiguration} from './cluster-types';
+import {formatDisplayNumber} from '../../shared/util/form-utils';
+import {Application, IAppConfiguration} from '../../shared/model/app.configuration.model';
+import {useAppSelector} from '../../config/store';
+import {emrApplications} from '../emr/emr.reducer';
+import {appConfig} from './configuration-reducer';
+
+export type EmrConfigurationProps = {
+    setFields: (any) => void;
+    touchFields: (any) => void;
+    expandedSections: {object};
+    setExpandedSections: (any) => void;
+    errors: any;
+    form: any;
+    maxInstances: number;
+    minInstances: number;
+    scaleOutIncrement: number;
+    scaleOutCooldown: number;
+    scaleOutPercentageMemAvailable: number;
+    scaleOutEvalPeriods: number;
+    scaleInIncrement: number;
+    scaleInCooldown: number;
+    scaleInPercentageMemAvailable: number;
+    scaleInEvalPeriods: number;
+};
+
+export function EmrConfiguration (props: EmrConfigurationProps) {
+    const applicationConfig: IAppConfiguration = useAppSelector(appConfig);
+    const emrApplicationList: string[] = useAppSelector(emrApplications);
+    const [selectedApplicationOptions, setSelectedApplicationOptions] = useState([] as any[]);
+    const [applicationOptions, setApplicationOptions] = useState([] as any);
+
+    useEffect(() => {
+        const appList: any[] = [];
```

Comment:
Looks like we're adding objects like `{value: string, label: string}` below, can we use that instead of `any`?

Reply:
Is this the wrong type or does this just not make sense to do?